app/presenters/content_item_presenter.rb

```diff
@@ -159,11 +159,19 @@ def facets
         name: "Document type",
         preposition: "of type",
         type: "text",
-        display_as_result_metadata: true,
+        display_as_result_metadata: false,
         filterable: true,
         allowed_values: detailed_format_allowed_values,
       },
       {
+        key: "display_type",
+        name: "Document type",
+        preposition: "of type",
```

Comment:
Does it need `type`? Some of the ones above (like `is_historic`) doesn't have type either...